client/src/components/Ticketing/ticketingmanager/TicketTypes/SubscriptionTypeUtils.tsx

```diff
@@ -0,0 +1,84 @@
+import {useEffect, useState} from 'react';
+import {getData} from '../Event/components/ShowingUtils';
+
+export const validateSubscriptionType = (subscriptionType: any) => {
+  if (+subscriptionType.price <= 0) {
+    throw new Error('Invalid price');
+  }
+  if (subscriptionType.description.length === '') {
+    throw new Error('Invalid Description');
+  }
+  if (subscriptionType.name.length === '') {
+    throw new Error('Invalid Name');
+  }
```

Comment:
For the errors that are displayed, I think it'd make sense for them to be a bit more descriptive. Like `Input Description` or `Please Input Description` or something around those lines. Also with the price, although it is an unlikely scenario, we probably don't want to have the arbitrary rule that they can't have a `$0` subscription price, so we'd just protect against negative prices.

Reply:
Updated